lucid/overview.py

```diff
@@ -358,7 +363,20 @@ def __init__(self, parent=None):
         self.stackUnder(self.overlay)
 
     def add_from_dict(self, items):
-        for location, dev_list in items.items():
+        rows = set()
+        cols = set()
+        for e in items:
+            r, c = e.split('|')
+            rows.add(r)
+            cols.add(c)
+
+        data = collections.OrderedDict()
+        for r in sorted(rows):
+            for c in sorted(cols):
+                entry = items.get(f"{r}|{c}", None)
+                data[f"{r}|{c}"] = entry if entry else []
```

Comment:
This is perfectly valid, but I've been on an `or` kick recently which makes things pretty concise and readable (in my opinion):

```suggestion
                data[f"{r}|{c}"] = items.get(f"{r}|{c}") or []
```

Reply:
I like it...